backend/src/models/connection.ts

```diff
@@ -2,6 +2,9 @@ import { createConnection, Connection } from 'typeorm';
 import {
   // Models for the Crossfeed database
   ApiKey,
+  SavedSearch,
+  Notification,
+  OrganizationTag,
```

Comment:
OrganizationTag is a duplicate import. It is also being imported at line 14.

Reply:
I went ahead and resolved the duplicate issue myself and rebuilt. The change can be viewed on commit: https://github.com/cisagov/XFD/commit/c3af8b7375a248e9be731b1a20f00e01d9e43b21